java/pattern/variable/Variable.java

```diff
@@ -94,7 +98,7 @@ public Reference reference() {
     }
 
     public boolean isNamed() {
-        return reference.isName();
+        return reference.isName() || reference().isValue();
```

Comment:
this is a bit awkward, we should discuss

Reply:
Let's try to remove `Label` from `Referable`, and rename this method to `isReferable`

you should try to see if this is easily propagated to core before we confirm the change. Plus it may not slide in Haikal's review